modules/jupyter-widget/src/create-deck.js

```diff
@@ -48,16 +55,19 @@ export function createDeck({
   handleClick,
   handleWarning
 }) {
+  let deckgl;
   try {
-    const jsonConverter = new deck.JSONConverter({
-      configuration: jsonConverterConfiguration
-    });
+    if (!jsonConverter) {
+      jsonConverter = new deck.JSONConverter({
```

Comment:
`JSONConverter` construction is very cheap, you can just define it with the variable declaration instead of on demand.